miio/tests/test_toiletlid.py

```diff
@@ -0,0 +1,94 @@
+from unittest import TestCase
+
+import pytest
+
+from miio.toiletlid import (
+    Toiletlid,
+    ToiletlidStatus,
+    AmbientLightColor,
+    MODEL_TOILETLID_V1,
+)
+from .dummies import DummyDevice
+
+"""
+Response instance
+>> status
+
+Work: False
+State: 1
+Ambient Light: Yellow
+Filter remaining: 100%
+Filter remaining time: 180
+"""
```

Comment:
Sorry for being unclear, I meant with my comment about the response for `get_prop` inside (documented under the state container, see e.g. https://github.com/rytilahti/python-miio/blob/master/miio/ceil.py#L20). Other than that this is ready to be merged :+1: